### DISCUSSION TITLE
I think we need showcase tab in VuejS docs

### DISCUSSION THREAD

**Discussion by JenuelDev** [vuejs/docs#2258] [Ideas]
For me, I am motivated by looking at what the framework can do. And one of this is the showcases. Looking at nextjs showcase I am motivated to learn reactjs instead. I believe VueJs docs should have a page for showcase to show some good apps made in vuejs.

**Comment by AloisSeckar** [comment]
There is such [section in Community Guide](https://vuejs.org/about/community-guide.html#explore-the-ecosystem)

However it is true, that it maybe deserves a link or two being uplifted somewhere, because it is not the part where would you look for it first.

**Comment by splincool** [comment]
There is a great site https://madewithvuejs.com/ with tons of cool app examples. Perhaps this can be added to the documentation website.
